module/modmanager/manager.go

```diff
@@ -1207,6 +1207,14 @@ func (m *module) startProcess(
 		m.logger.CInfow(ctx, "Starting module in working directory", "module", m.cfg.Name, "dir", moduleWorkingDirectory)
 	}
 
+	// Create STDOUT and STDERR loggers for the module and turn off log deduplication for
+	// both. Module output through these loggers may contain data like stack traces, which
+	// are repetitive but are not actually "noisy."
+	stdoutLogger := m.logger.Sublogger("StdOut")
+	stderrLogger := m.logger.Sublogger("StdErr")
+	stdoutLogger.NeverDeduplicate()
+	stderrLogger.NeverDeduplicate()
```

Comment:
Manually tested that STDOUT and STDERR output from the simple module example does _not_ get deduplicated with these changes where it _would_ have been deduplicated before.